From 102fbfd31dd26c911e0b5313e3e36ded8a3d390a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ji=C5=99=C3=AD=20Kone=C4=8Dn=C3=BD?=
 <DragonLichcz@gmail.com>
Date: Thu, 12 Nov 2015 19:32:48 +0100
Subject: [PATCH] Updated desktop file for the new format

Now it pass without warning or error through desktop-file-validate
tester.
---
 etc/PlayOnLinux.desktop | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/etc/PlayOnLinux.desktop b/etc/PlayOnLinux.desktop
index 08b1d23..76b5bfa 100644
--- a/etc/PlayOnLinux.desktop
+++ b/etc/PlayOnLinux.desktop
@@ -1,9 +1,8 @@
 [Desktop Entry]
 Version=1.0
-Encoding=UTF-8
 Name=PlayOnLinux
-Comment=PlayOnLinux
-Type=Application %F
-Exec=playonlinux
+Comment=Front-end application for the wine
+Type=Application
+Exec=playonlinux %F
 Icon=/usr/share/playonlinux/etc/playonlinux.png
-Categories=Application;Game
+Categories=Utility;Emulator;
-- 
2.7.4

